README.md

```diff
@@ -39,54 +39,38 @@ ant test
 
 See the [documentation for api usage].
 
-After a user completes the oauth2 flow, the credentials for their account will be stored as a data source with a configured folder created in the user's root mailbox during authentication. If an import class is associated, it can be triggered manually with a `zmsoap` import data request.
+After a user completes the oauth2 flow, the credentials for their account will be stored as a data source with a configured folder created in the user's root mailbox during authentication. An import can be triggered manually with a `zmsoap` import data request.
```

Comment:
`An import can be triggered manually with a `zmsoap` import data request.` - This should be reworded now since import is triggered after oauth flow.